### PR TITLE
Add io.github.alexkdeveloper.zodiac

### PR DESCRIPTION
**Pull request by alexkdeveloper**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[maint]: https://docs.flathub.org/docs/for-app-authors/maintanance
[submission]: https://docs.flathub.org/docs/for-app-authors/submission

### CODE REVIEW

**Comment by alexkdeveloper** [comment]
Bot, build io.github.alexkdeveloper.zodiac

**Comment by alexkdeveloper** [comment]
Bot, build io.github.alexkdeveloper.zodiac

**Comment by alexkdeveloper** [comment]
Sorry.
Perhaps the first build should be from the administrator.
I didn't know that)

**Comment by hfiguiere** [comment]
no. It's just you keep writting `Bot`. So it doesn't understand.

**Comment by alexkdeveloper** [comment]
bot, build io.github.alexkdeveloper.zodiac

**Comment by alexkdeveloper** [comment]
bot, build io.github.alexkdeveloper.zodiac

**Comment by alexkdeveloper** [comment]
bot, build io.github.alexkdeveloper.zodiac

File: io.github.alexkdeveloper.zodiac.json [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+},
+        {
+            "name" : "Zodiac",
+            "builddir" : true,
+            "buildsystem" : "meson",
+            "sources" : [
+                {
+                    "type": "git",
+                    "url": "https://github.com/alexkdeveloper/zodiac.git",
+                    "tag": "1.0.0"
```
Comment (hfiguiere) [inline io.github.alexkdeveloper.zodiac.json:208]: You should add a commit to protect against retagging.

**Comment by alexkdeveloper** [comment]
bot, build io.github.alexkdeveloper.zodiac

**Pull request closed without merging**